### PR TITLE
Close guests popup on save

### PR DESCRIPTION
**Pull request by tbergq**
---
Please check this before merging (do not delete this checklist):

- [ ] it works in landscape and portrait mode
- [ ] it uses `cacheConfig.force=true` if offline access doesn't make sense

### CODE REVIEW

File: app/hotels/src/allHotels/searchForm/guests/__tests__/GuestsPopup.test.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    const renderer = shallowRenderGuestsPopup(jest.fn());
+    expect(renderer).toMatchSnapshot();
+  });
+
+  it('should call onClose from handleSave when child age is not missing', async () => {
+    const onClose = jest.fn();
+    const testRenderer = renderGuestsPopup(onClose);
+    const testInstance = testRenderer.root;
+
+    await testInstance.instance.handleSave();
```
Comment (mrtnzlml) [inline app/hotels/src/allHotels/searchForm/guests/__tests__/GuestsPopup.test.js]: Why are you awaiting it? It is not a promise...

File: app/hotels/src/allHotels/searchForm/guests/__tests__/GuestsPopup.test.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  it('should not call onClose from handleSave when child age is missing', async () => {
+    const onClose = jest.fn();
+    const testRenderer = renderGuestsPopup(onClose);
+    const testInstance = testRenderer.root;
+
+    const isAgeMissingMock = jest.fn();
+    isAgeMissingMock.mockReturnValue(true);
+    testInstance.instance.isMissingAge = isAgeMissingMock;
+
+    await testInstance.instance.handleSave();
```
Comment (mrtnzlml) [inline app/hotels/src/allHotels/searchForm/guests/__tests__/GuestsPopup.test.js]: Same here.

File: app/hotels/src/allHotels/searchForm/guests/__tests__/GuestsPopup.test.js (hunk 1 above)
Comment (tbergq) [reply app/hotels/src/allHotels/searchForm/guests/__tests__/GuestsPopup.test.js]: Removed await

File: app/hotels/src/allHotels/searchForm/guests/__tests__/GuestsPopup.test.js (hunk 2 above)
Comment (tbergq) [reply app/hotels/src/allHotels/searchForm/guests/__tests__/GuestsPopup.test.js]: Removed await

**Pull request merged**